src/routes/note-edit.test.js

```diff
@@ -0,0 +1,76 @@
+import { MemoryRouter } from "react-router-dom";
+import { render, cleanup, act, fireEvent } from '@testing-library/react';
```

Comment:
When we destructuring more then one element from the file, we write each variable on separate string, please, refactor it.

```suggestion
import {
  render,
  cleanup,
  act,
  fireEvent,
} from '@testing-library/react';
```